src/transformers/utils/quantization_config.py

```diff
@@ -1309,7 +1309,8 @@ def get_apply_tensor_subclass(self):
         return _STR_TO_METHOD[self.quant_type](**self.quant_type_kwargs)
 
     def __repr__(self):
-        return f"{self.quant_type}({', '.join(str(k) + '=' + str(v) for k, v in self.kwargs.items())})"
+        config_dict = self.to_dict()
+        return f"{self.__class__.__name__} {json.dumps(config_dict, indent=2, sort_keys=True)}\n"
```

Comment:
Also, this is the tiniest nit ever, but do reprs normally end in `\n`?

Reply:
I agree that it looks unnecessary, but I copied this 1:1 from here for consistency:

https://github.com/huggingface/transformers/blob/e2ac16b28a0b8b900e136750309ca40c49d975c5/src/transformers/utils/quantization_config.py#L285-L287